.circleci/src/jobs/build_ios.yml

```diff
@@ -1,5 +1,5 @@
 macos:
-  xcode: 14.3.1
+  xcode: 15.2.0
```

Comment:
See this conversation: https://github.com/digitalfabrik/entitlementcard/pull/1248/files#r1449125363

Please check this upgrade does not break the CD

Reply:
yes thats why i skip 15.0.1 :)